web/src/components/home/faq.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+import Button from '../shared/button';
+import Accordion from '../shared/accordion/accordion';
+import { AlignedCenterContainer } from '../../config/global-styled-components';
+import { StyledH2 } from '../../config/global-styled-components';
+import FaqJson from '../../config/faq-conf.json';
+import { device } from '../../config/device';
+
+const FaqContainer = styled(AlignedCenterContainer)`
+  margin-top: 5rem;
```

Comment:
Remember that large ui is made with a  max-width: 1600px with a margin:auto (so it gets centered) on top of the padding. Check how was done the shipping-info or the product categories. this should also be fixed on the favorites carousel, you can decide if you want to fix it in this pull request or the next one 😄